networkit/centrality.pyx

```diff
@@ -2640,6 +2640,58 @@ cdef class ApproxElectricalCloseness(Centrality):
 		return (<_ApproxElectricalCloseness*>self._this).computeExactDiagonal(tol)
 
 
+cdef extern from "<networkit/centrality/ForestCentrality.hpp>":
+	cdef cppclass _ForestCentrality "NetworKit::ForestCentrality"(_Centrality):
+		_ForestCentrality(_Graph G, node root, double eps, double kappa) except +
+		vector[double] &getDiagonal() except +
+
+cdef class ForestCentrality(Centrality):
+	"""
+	ForestCentrality(G, root, eps = 0.1, kappa = 0.3)
```

Comment:
Currently we (mostly) use default variable declaration without whitespace in documentation strings.